There's no way this failure is related, right?

```
======================================================================
FAIL: testRepeatedFailedLogin (auditcare.tests.auth.authenticationTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "submodules/auditcare-src/auditcare/tests/auth.py", line 115, in testRepeatedFailedLogin
    self.assertEquals(firstlogin_count, next_count)
AssertionError: 1 != 2
----------------------------------------------------------------------
```

--- a/corehq/util/timezones/utils.py
+++ b/corehq/util/timezones/utils.py
@@ -28,7 +28,7 @@ def get_timezone_for_request(request=None):
     domain = getattr(request, 'domain', None)
 
     if user or domain:
-        return get_timezone_for_user(request.couch_user, request.domain)
+        return get_timezone_for_user(user, domain)
     else:
         return None
 

